kubejs/server_scripts/expert/recipes/enigmatica/dust_crushing_tiers.js

```diff
@@ -0,0 +1,176 @@
+ServerEvents.recipes((event) => {
+    if (global.isExpertMode == false) {
+        return;
+    }
+
+    const id_prefix = 'enigmatica:expert/enigmatica/';
+    // Tiers range from 1-4, with 4 being the highest
+    const recipes = [
+        {
+            input_types: ['gems'],
+            material: 'ender_pearl',
+            output: 'thermal:ender_pearl_dust',
```

Comment:
Is this the ender pearl dust variant we want to use?

Reply:
Fair point, I've added it to EE now.